public/components/datasources/components/manage/accelerations/acceleration_details_flyout.tsx

```diff
@@ -37,16 +37,25 @@ export interface AccelerationDetailsFlyoutProps {
   dataSourceMDSId?: string;
 }
 
-const getMappings = (index: string): Promise<OpenSearchDashboardsResponse> | undefined => {
-  return coreRefs.dslService?.fetchFields(index);
+const getMappings = (
+  index: string,
+  dataSourceMDSId?: string
+): Promise<OpenSearchDashboardsResponse> | undefined => {
```

Comment:
return Promise or undefined? is it supposed to be Promise<OpenSearchDashboardsResponse | undefined>?